README.md

```diff
@@ -4,24 +4,25 @@
 <a href="https://github.com/camaraproject/§repo_name§/graphs/contributors" title="Contributors"><img src="https://img.shields.io/github/contributors/camaraproject/§repo_name§?style=plastic"></a>
 <a href="https://github.com/camaraproject/§repo_name§" title="Repo Size"><img src="https://img.shields.io/github/repo-size/camaraproject/§repo_name§?style=plastic"></a>
 <a href="https://github.com/camaraproject/§repo_name§/blob/main/LICENSE" title="License"><img src="https://img.shields.io/badge/License-Apache%202.0-green.svg?style=plastic"></a>
+<a href="https://github.com/camaraproject/§repo_name§/releases/latest" title="Latest Release"><img src="https://img.shields.io/github/release/camaraproject/§repo_name§?style=plastic"></a>
 
 # §repo_name§
-Repository to describe, develop, document and test the §repo_name§ API family
+Repository to describe, develop, document, and test the §repo_name§ API family
 
 ## Scope
 * Service APIs for “§repo_name§” (see APIBacklog.md)  
 * It provides the customer with the ability to:  
   * §API family description§.
-  * NOTE: The scope of this API family should be limited (at least at a first stage) to 4G and 5G.  
-* Describe, develop, document and test the APIs (with 1-2 Telcos)  
+  * NOTE: The scope of this API family should be limited (at least at the first stage) to 4G and 5G.  
+* Describe, develop, document, and test the APIs (with 1-2 Telcos)  
 * Started: §start date§
-* Location: virtually  
+* [Latest Release](https://github.com/camaraproject/§repo_name§/releases/latest)
```

Comment:
At the start of a sub project there is no latest release. To be more precise: with the new versioning proposal of Release Management there will be for a longer time only pre-releases. See my proposal for the section "Status and released versions"

Reply:
PS: the line should here be deleted, it does not belong to the Scope.

Reply:
In some of the other repo readmes, we moved the "latest release section" to the Contributing section below the mailing list.

Reply:
The information about the latest release isn't meant in first place for contributor (they contribute to main), but for people from outside, interested in the latest version for their work.

Reply:
Please consider my proposal for the "Status and released version" section.